reservation/test_typeof_job_model.py

```diff
@@ -13,8 +13,8 @@ def persisted_typeOfJob(typeOfJob):
 
 
 @pytest.fixture
-def persisted_type_of_job_pool(persisted_typeOfJob, professional):
-    typeOfJob_object = TypeOfJob(professional_id=professional, typeOfJob_name="Hair cut", price=100)
+def persisted_type_of_job_pool(persisted_typeOfJob, make_typeOfJob):
+    typeOfJob_object = make_typeOfJob()
```

Comment:
I think you can call it twice with different parameters in order to to have 
typeOfJob_object and persisted_typeOfJob, WDYT? 

In general I do not see the use case of persistent and not persistent data, in tests,
Isnt it persistent all the way?

Reply:
Done